ginga/AstroImage.py

```diff
@@ -139,8 +139,7 @@ def load_file(self, filepath, numhdu=None, naxispath=None):
             if name is None:
                 name = info.name
                 if ('[' not in name):
-                    if numhdu is None:
-                        numhdu = numhdu_
+                    numhdu = numhdu_
```

Comment:
This is the only change that worries me. Because you are overriding the `numhdu` that the user specified.  Since this is an important method that can be called from many places, the user may want to force the use of a specific `numhdu`.

To preserve the possibility of forcing a specific `numhdu`, but defaulting to an override, how about adding another kwarg parameter `allow_numhdu_override` which defaults to `True`.  Then the logic looks like:
```python
if (numhdu is None) and allow_numhdu_override:
    numhdu = numhdu_
```
That way it will work in the default with the reference viewer, but if someone has a special use case with their custom viewer they can always force the `numhdu`.